src/components/app/details/appConfig/AppConfig.tsx

```diff
@@ -358,6 +359,14 @@ export default function AppConfig({ appName, isJobView, filteredEnvIds }: AppCon
             : ''
     }
 
+    function handleSaveButton() {
+        gitOpsConfigDevtron(+appId, repositoryURL).then((response)=>{
+            console.log(response)
```

Comment:
Why console.log? Should we not show any loader and success toast?

Reply:
This API call is handled in Mukul's PR so removed it from here.